src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -90,6 +92,19 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
         THROW_LAST_ERROR_IF_NULL(_hOutputThread);
 
+        STARTUPINFO si = { 0 };
+        si.cb = sizeof(STARTUPINFOW);
+
+        // If we have a startingTitle, create a mutable character buffer to add
+        // it to the STARTUPINFO.
+        std::unique_ptr<wchar_t[]> mutableTitle = std::make_unique<wchar_t[]>(_startingTitle.size() + 1);

```

Comment:
we'll allocate one byte and then discard it later if the user doesn't set a starting title.

Reply:
This remains true even in the latest edition.